.github/workflows/ci.yml

```
@@ -42,14 +81,78 @@ jobs:
+  bugbot-context:
+    name: Cursor BugBot Context
+    runs-on: windows-latest
+    needs: build
+    if: github.event_name == 'pull_request'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Prepare context
+        shell: bash
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:4:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)